packages/frontend/src/components/ElectionForm/ElectionSettings.tsx

```diff
@@ -74,11 +80,10 @@ export default function ElectionSettings() {
         }
         const success = await updateElection(election => {
             election.settings = editedElectionSettings
-            election.is_public = editedIsPublic
         })
         if (!success) return false
         await refreshElection()
-        handleClose()
```

Comment:
Why was this removed? Wouldn't syncState still need to be called as a side effect?